beacon-chain/sync/validate_sync_committee_message_test.go

```diff
@@ -588,7 +588,7 @@ func TestValidateSyncCommitteeMessage_Optimistic(t *testing.T) {
 			Topic: &topic,
 		},
 	}
-	res, err := r.validateCommitteeIndexBeaconAttestation(ctx, "foobar", msg)
+	res, err := r.validateSyncCommitteeMessage(ctx, "foobar", msg)
```

Comment:
These tests are bad to begin with. `TestValidateSyncCommitteeMessage_Optimistic` is calling  `validateCommitteeIndexBeaconAttestation`. I fixed it here but they will be removed in #11320